esrally/rally.py

```diff
@@ -1024,9 +1072,9 @@ def dispatch_sub_command(arg_parser, args, cfg):
             cfg.add(config.Scope.applicationOverride, "system", "list.config.option", args.configuration)
             cfg.add(config.Scope.applicationOverride, "system", "list.max_results", args.limit)
             cfg.add(config.Scope.applicationOverride, "system", "admin.track", args.track)
-            cfg.add(config.Scope.applicationOverride, "system", "list.races.benchmark_name", args.benchmark_name)
+            cfg.add(config.Scope.applicationOverride, "system", "list.benchmark_name", args.benchmark_name)
             cfg.add(config.Scope.applicationOverride, "system", "list.from_date", args.from_date)
-            cfg.add(config.Scope.applicationOverride, "system", "list.races.to_date", args.to_date)
+            cfg.add(config.Scope.applicationOverride, "system", "list.to_date", args.to_date)
```

Comment:
Did you really have to change this? to_date and benchmark_name are only used for races anyway. (Maybe we can keep `list.to_date` for consistency with `list.from_date` but keeping `list.races.benchmark_name` makes more sense to me.)